ethcore/res/ethereum/kovan_wasm_test.json

```diff
@@ -0,0 +1,74 @@
+{
+	"name": "Kovan",
+	"dataDir": "kovan",
+	"engine": {
+		"authorityRound": {
+			"params": {
+			"stepDuration": "4",
+			"blockReward": "0x4563918244F40000",
+				"validators" : {
+					"list": [
+						"0x00D6Cc1BA9cf89BD2e58009741f4F7325BAdc0ED",
+						"0x00427feae2419c15b89d1c21af10d1b6650a4d3d",
+						"0x4Ed9B08e6354C70fE6F8CB0411b0d3246b424d6c",
+						"0x0020ee4Be0e2027d76603cB751eE069519bA81A1",
+						"0x0010f94b296a852aaac52ea6c5ac72e03afd032d",
+						"0x007733a1FE69CF3f2CF989F81C7b4cAc1693387A",
+						"0x00E6d2b931F55a3f1701c7389d592a7778897879",
+						"0x00e4a10650e5a6D6001C38ff8E64F97016a1645c",
+						"0x00a0a24b9f0e5ec7aa4c7389b8302fd0123194de"
+					]
+				},
+				"validateScoreTransition": 1000000,
+				"validateStepTransition": 1500000,
+				"maximumUncleCountTransition": 5067000,
+				"maximumUncleCount": 0
+			}
+		}
+	},
+	"params": {
+		"gasLimitBoundDivisor": "0x400",
+		"registrar" : "0xfAb104398BBefbd47752E7702D9fE23047E1Bca3",
+		"maximumExtraDataSize": "0x20",
+		"minGasLimit": "0x1388",
+		"networkID" : "0x2A",
+		"forkBlock": 4297256,
+		"forkCanonHash": "0x0a66d93c2f727dca618fabaf70c39b37018c73d78b939d8b11efbbd09034778f",
+		"validateReceiptsTransition" : 1000000,
+		"eip155Transition": 1000000,
+		"validateChainIdTransition": 1000000,
+		"eip140Transition": 5067000,
+		"eip211Transition": 5067000,
+		"eip214Transition": 5067000,
+		"eip658Transition": 5067000,
+		"wasmActivation": 10
```

Comment:
The convention seems to be `*Transition`, would keep it.

Reply:
"wasmTransition" might imply that evm no longer works, i think it's not the best convention in this case

Reply:
Maybe `enableWasmTransition` or `supportWasmTransition` then?

Reply:
both `enableWasmTransition` and `supportWasmTransition` implies the possibilty of such transition at all, not some blocknumber where it is activated

it is not really about transition as eip-s

Reply:
`supportWasmTransition` and `enableWasmTransition` implies for me that `wasm is supported` or `wasm is enabled` starting at some block. Similar to how `eip658Transition` implies that `eip658` is active at some point (it doesn't disable any previous eips).
That said my main reason is to complain is to keep the naming consistent here, if in the future we support a fork to wasm-only network we can either have: `wasmOnlyTransition` or `disableEvmTransition` there. Introducing another convention will make the file even messier imho.